gramine/Dockerfile

```diff
@@ -0,0 +1,45 @@
+## CI/CD version
+
+###
+### TODO Get JAR from native image instead of rebuilding everything
+### Build application jar
+###
+FROM gradle:6.8.3-jdk11 as gradle-builder
+ARG HOME=/home
+WORKDIR $HOME
+COPY build.gradle gradle.properties settings.gradle  ./
+COPY src/ src/
+RUN gradle build -i --no-daemon
+RUN ls -al /home/build/libs
+
+
+###
+### Build the final graminized image with
+### the produced jar.
+###
+FROM iexechub/iexec-graphene-base:0.4.0
+
+COPY --from=gradle-builder /home/build/libs/app-all.jar /workplace/app/app.jar
+
+RUN apt update &&  apt-get install -y openjdk-11-jdk
```

Comment:
A best practice is to clean apt caches after installing packages in a **Dockerfile RUN statement**
See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Reply:
To keep things minimal, `openjdk-11-jdk-headless` or even `openjdk11-jre-headless` should be sufficient.

Reply:
Done in https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/54/commits/40daed62d2f7e0ba69fb4ca1b8357abf42d230ba :)

Reply:
Sorry for the typo this morning, thanks for the improvement